### DISCUSSION TITLE
猫耳FM-每日鱼干 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#15] [模板讨论]
猫耳FM-每日鱼干 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by miaoxiaozhao** [comment]
Failed at 1/1 request,
Fail assert: {"re": "\"success\":false", "from": "content"} from failed_asserts,
Request URL: https://www.missevan.com/member/getcatears?gtype=1

更新了CK也不行了。

**Reply by wjf0214** [reply]
我看我这可以啊。。没改过。。
